hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Listing.java

```diff
@@ -18,6 +18,7 @@
 
 package org.apache.hadoop.fs.s3a;
 
+import org.apache.hadoop.fs.s3a.api.S3ObjectStorageClassFilter;
```

Comment:
new org.apache imports MUST go into that block for new classes. set your IDE up for this and life gets simpler for all